src/main/resources/assets/seedmapper/lang/de_de.json

```diff
@@ -23,12 +28,30 @@
     "command.config.listBlocks": "Aktuell ignoriert: %s.",
     "command.config.listBlocks.empty": "Aktuell werden keine Blöcke ignoriert.",
 
+    "command.highlight.block.noneFound": "Kein %s zum hervorheben konnte gefunden werden.",
+    "command.highlight.block.success": "%d %s wurde hervorgehoben.",
+
+    "command.locate.biome.noneFound": "Konnte %s innerhalb 6400 Blöcken nicht finden.",
+    "command.locate.biome.success.0": "%s gefunden bei ",
+    "command.locate.biome.success.1": "Klicke um die Koordinaten von %s zu kopieren.",
+    "command.locate.feature.structure.noneFound": "Konnte %s innerhalb 6400 Blöcken nicht finden.",
+    "command.locate.feature.structure.success.0": "%s gefunden bei ",
+    "command.locate.feature.structure.success.1": "Klicke um die Koordinaten von %s zu kopieren.",
+    "command.locate.feature.slimeChunk.noneFound": "Konnte Slime Chunks innerhalb 6400 Blöcken nicht finden.",
```

Comment:
I think in the german translations there is always a `von` missing in front of the `6400 Blöcken`.
So it should be `innerhalb von 6400Blöcken nicht finden.`
Not  `innerhalb 6400 Blöcken nicht finden.`

Reply:
good call...thank you